module/VuFind/src/VuFind/ILS/Driver/Sierra.php

```diff
@@ -107,7 +107,10 @@ protected function getIds($id)
             . "FROM sierra_view.bib_view "
             . "LEFT JOIN sierra_view.bib_record_item_record_link ON "
             . "(bib_view.id = bib_record_item_record_link.bib_record_id) "
-            . "WHERE bib_view.record_num = $1;";
+            . "INNER JOIN sierra_view.item_view ON "
+            . "(bib_record_item_record_link.item_record_id = item_view.id "
```

Comment:
Am I misreading something, or is there a mismatched parenthesis here? Please let me know whether or not this is cause for concern -- just wanted to be totally sure before merging, since I can't test this from here. Thanks!